test/functional/p2p-segwit.py

```diff
@@ -1944,7 +1944,7 @@ def run_test(self):
         self.test_signature_version_1()
         self.test_non_standard_witness()
         sync_blocks(self.nodes)
-        self.test_upgrade_after_activation(self.nodes[2], 2)
+        self.test_upgrade_after_activation(2)
```

Comment:
Please name primitive types. If you don't want to clutter the scope with variable names, you could always just use named args.